x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_view/index.tsx

```diff
@@ -55,6 +56,14 @@ export function StreamDetailViewContent({ name, tab }: { name: string; tab: stri
       }),
       background: true,
     },
+    {
```

Comment:
let's remove this entirely for now, or comment it out, until we actually have the UI